packages/js-utils/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## 1.0.0-alpha.0
```

Comment:
/nit: is not very common to find packages with `-alpha` or similar. Something like `0.1.0` or `1.0.0` feels more natural to me.

Reply:
fixed in https://github.com/Automattic/wp-calypso/pull/50263/commits/b9db7131f5b71d1159efdb1f06ab7fd89eda46f2, I don't think there's a need to maintain an alpha version